Makefile

```diff
@@ -92,6 +94,21 @@ TRIVY_VERSION ?= 0.14.0
 PROTOC_VERSION ?= 3.15.2
 SHELLCHECK_VER ?= v0.7.2
 
+
+# For aws integration tests 
+BUILD_TIMESTAMP_W_SEC := $(shell date +%Y-%m-%d-%H-%M-%S)
+EKS_CLUSTER_NAME := integ-cluster-$(BUILD_TIMESTAMP_W_SEC)
+AWS_SERVICE_ACCOUNT_NAME := basic-test-mount-sa
```

Comment:
Where is this being used?

Reply:
Could these defaults be set in `bats` file or the script?

Reply:
EKS_CLUSTER_NAME  is needed for the make targets setup-eks-cluster and e2e-eks-cleanup

AWS_SERVICE_ACCOUNT_NAME was required to be set in the Makefile when I had cluster creation in the Makefile but since I moved cluster creation to the script, it is not required in the Makefile anymore. Will remove